web/ui/static/js/targets.js

```diff
@@ -21,6 +21,15 @@ function showUnhealthy(_, container) {
 }
 
 function init() {
+  if (!localStorage.selectedTab || localStorage.selectedTab == "all-targets"){
+    $("div label:nth-child(1)").addClass("active");
+    $(".table-container").each(showAll);
+  }
+  else if (localStorage.selectedTab == "unhealthy-targets") {
```

Comment:
Style-wise, could you put this on the same line as the brace above it? (`} else if (...`).

Reply:
Done.